util/PostgresMigrations/Scripts/2022-09-26_00_SmServiceAccount.psql

```diff
@@ -0,0 +1,18 @@
+START TRANSACTION;
+
+CREATE TABLE "ServiceAccount" (
+    "Id" uuid NOT NULL,
+    "OrganizationId" uuid NOT NULL,
+    "Name" text NULL,
+    "CreationDate" timestamp without time zone NOT NULL,
+    "RevisionDate" timestamp without time zone NOT NULL,
+    CONSTRAINT "PK_ServiceAccount" PRIMARY KEY ("Id"),
+    CONSTRAINT "FK_ServiceAccount_Organization_OrganizationId" FOREIGN KEY ("OrganizationId") REFERENCES "Organization" ("Id") ON DELETE CASCADE
+);
+
+CREATE INDEX "IX_ServiceAccount_OrganizationId" ON "ServiceAccount" ("OrganizationId");
+
+INSERT INTO "__EFMigrationsHistory" ("MigrationId", "ProductVersion")
+VALUES ('20220926190851_SmServiceAccount', '6.0.4');
+
+COMMIT;
```

Comment:
```suggestion
COMMIT;

```

Reply:
Resolved in https://github.com/bitwarden/server/pull/2301/commits/e2746d040e644a4550ba4d15595dabc375a8adf8